package.json

```diff
@@ -32,6 +33,11 @@
     "@types/react-dom": "~18.2.18",
     "eslint": "^8",
     "eslint-config-next": "14.1.0",
+    "eslint-config-prettier": "^9.1.0",
+    "husky": "^9.0.7",
+    "lint-staged": "^15.2.1",
+    "prettier": "3.2.4",
+    "prettier-plugin-prisma": "^5.0.0",
```

Comment:
I believe will be better to put linting dependencies to the devDependencies chapter.